pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -77,7 +77,7 @@ func TestUserSignupCreateMUROk(t *testing.T) {
 	require.Len(t, murs.Items, 1)
 	mur := murs.Items[0]
 	require.Equal(t, test.HostOperatorNs, mur.Namespace)
-	require.Equal(t, userSignup.Name, mur.Labels[v1alpha1.MasterUserRecordUserIDLabelKey])
+	require.Equal(t, userSignup.Name, mur.Labels[v1alpha1.MasterUserRecordOwnerLabelKey])
```

Comment:
There are still many cases where the `MasterUserRecordUserIDLabelKey` is used in this test, but you don't change it to `MasterUserRecordOwnerLabelKey` nor add a check for the existence of the new label. I know that these are the cases for already existing MURs, but I believe that it would be good to migrate all to the new label, wouldn't it?

Reply:
I must have missed these extra instances, hopefully they're all covered now.